packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -12,14 +12,44 @@ const DEFAULT_SCHEDULE = {
   cron: '0 * * * * *',
 }
 
-const validateRootTask = (log, props) =>
-  expect(log).toMatchSnapshot({
+// it only support one schedule
+const validateBackupJob = async (xo, input, output) => {
```

Comment:
I'm not found of async validation methods, for instance, you forgot to `await` it in your first test.

Reply:
Yes my bad, need `await`s in the tests